packages/@react-spectrum/s2/src/style-utils.ts

```diff
@@ -113,7 +113,8 @@ export const fieldInput = () => ({
   contain: {
     // Only apply size containment if contain-intrinsic-width is supported.
     // In older browsers, this will fall back to the default browser intrinsic width.
-    '@supports (contain-intrinsic-width: 1px)': 'inline-size'
+    '@supports (contain-intrinsic-width: 1px)': 'inline-size',
+    isQuiet: 'none'
```

Comment:
honestly not too sure about this but without it, there's too much spacing between the value text and the dropdown chevron which doesn't match the figma designs